docs/drafts/derive-serviceaccount.md

```diff
@@ -0,0 +1,401 @@
+# Derive minimal ServiceAccount required for ClusterExtension Installation and Management
+
+OLMv1 does not provide cluster admin privileges by default for installing cluster extensions. This means that the installation process will require a service account with sufficient privileges to install the bundle. It depends on the cluster extension developer to specify the exact permissions required for the management of any specific bundle. A Service Account needs to be explicitly specified for installing and upgrading operators else will face errors when deploying your cluster extension.
```

Comment:
So, if you are referring to the specific API object, use the camel case "ServiceAccounts". If you are referring to it more generically, use "service accounts". 

Since we don't have a specific style guide to work off of right now, I would say that either "service account" or "ServiceAccount" is fine. We can always make a sweep after we come up with some basic style rules and set up some supporting tooling around it.

Reply:
i updated the two occurences from `Service Account` to `ServiceAccount`